ZenPacks/zenoss/ZenPackLib/lib/spec/ZenPackSpec.py

```diff
@@ -466,6 +482,25 @@ def create_device_js_snippet(self):
         return self.create_js_snippet(
             'device', snippet, classes=device_classes)
 
+    def create_add_js_snippet(self):
+        """Register component add JavaScript snippet."""
+        device_classes = [
```

Comment:
Treating all device types within the same ZenPack to the same components on their add menu will probably end up getting us in trouble. Such as in the Windows ZenPack where we have Device and ClusterDevice, or someday in cases like CiscoUCS where we have Device and CIMCDevice.